Composer/packages/ui-plugins/composer/src/defaultFlowSchema.ts

```diff
@@ -0,0 +1,195 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Will we want to move this over to the sdk component schemas in this release?

Reply:
yes we do, plan to do that in another PR